Dockerfile

```diff
@@ -0,0 +1,45 @@
+# common image to install CentOS7, updates, needed packages and NPM
+FROM centos:7
+
+# install Python 3.5
+RUN yum install -y https://centos7.iuscommunity.org/ius-release.rpm
+RUN yum install -y \
+        python35u \
+        python35u-devel \
+        python35u-pip \
+        yum-utils \
+        curl \
+        git \
+        rlwrap \
+        screen \
+        vim \
+        emacs-nox && \
+    yum install -y \
+        epel-release && \
+    yum groupinstall -y "Development Tools" && \
+    yum install -y \
+        libffi-devel \
+        libxml2-devel \
+        libxslt-devel \
+        npm
+
+RUN yum clean -y all
+
+# set python3.5 as default
+RUN alternatives --install /usr/bin/python python /usr/bin/python2 50
+RUN alternatives --install /usr/bin/python python /usr/bin/python3.5 60
+RUN alternatives --set python /usr/bin/python3.5
```

Comment:
Could we install Python3.6? I have tested it using python3.5 it works fine, there are [some parts](https://github.com/inveniosoftware/cookiecutter-invenio-instance/pull/73/files#diff-a902f45571aa281de04160261c58f343R4) that have to be moved here, we can see together :)